.github/workflows/bump_release_version.yml

```diff
@@ -0,0 +1,30 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump version"
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  bump:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Modify File
+        run: |
+          RELEASE_VERSION=$(git describe --abbrev=0 | cut -d '-' -f 2)
+         ./.ci/bump-release-version.sh "$RELEASE_VERSION-SNAPSHOT"
+      - name: GitHub Milestone
+        id: MILESTONE
```

Comment:
why you need `id` ? remove it, as it is not present in other places

Reply:
Changes Done